cookbook/recipes/subscriptions/recipe.yaml

```diff
@@ -0,0 +1,111 @@
+deletedFiles: []
+steps:
+  - type: INFO
+    name: Set up the Shopify Subscriptions app
+    description: >
+      1. Install the [Shopify Subscriptions
+      app](https://apps.shopify.com/shopify-subscriptions).
+
+      2. In your Shopify admin, use the Subscriptions app to create one or more
+      subscription plans.
+
+      3. Open the product details pages for any products that will be sold as
+      subscriptions, and add the relevant subscription plans in the **Purchase
+      options** section.
+
+
+      The Hydrogen demo storefront comes pre-configured with an example
+      subscription product with the handle `shopify-wax`.
+  - type: COPY_INGREDIENTS
+    name: Add ingredients to your project
+    description: Copy the ingredients from the `ingredients/` directory to the
+      current directory.
+    ingredients:
+      - templates/skeleton/app/components/SellingPlanSelector.tsx
+      - templates/skeleton/app/styles/selling-plan.css
+  - type: PATCH
+    name: Render the selling plan in the cart
+    description: >
+      1. Update `CartLineItem` to show subscription details when they're
+      available.
+
+      2. Extract `sellingPlanAllocation` from cart line data, display the plan
+      name, and standardize component import paths.
+    diffs:
+      - file: app/components/CartLineItem.tsx
+        patchFile: CartLineItem.tsx.8e657b.patch
+  - type: PATCH
+    name: Update `ProductForm` to support subscriptions
+    description: >
+      1. Add conditional rendering to display either subscription options or
+      standard variant selectors.
+
+      2. Implement `SellingPlanSelector` and `SellingPlanGroup` components to
+      handle subscription plan selection.
+
+      3. Update `AddToCartButton` to include selling plan data when
+      subscriptions are selected.
+    diffs:
+      - file: app/components/ProductForm.tsx
+        patchFile: ProductForm.tsx.8e409a.patch
+  - type: PATCH
+    name: Update `ProductPrice` to display subscription pricing
+    description: >
+      1. Add a `SellingPlanPrice` function to calculate adjusted prices based on
+      subscription plan type (fixed amount, fixed price, or percentage).
+
+      2. Add logic to handle different price adjustment types and render the
+      appropriate subscription price when a selling plan is selected.
+    diffs:
+      - file: app/components/ProductPrice.tsx
+        patchFile: ProductPrice.tsx.a5e47f.patch
+  - type: PATCH
+    name: Add selling plan data to cart queries
+    description: >
+      Add a `sellingPlanAllocation` field with the plan name to the standard and
+      componentizable cart line GraphQL fragments. This displays subscription
+      details in the cart.
+    diffs:
+      - file: app/lib/fragments.ts
+        patchFile: fragments.ts.e8eb04.patch
+  - type: PATCH
+    name: Add `SellingPlanSelector` to product pages
+    description: >
+      1. Add the `SellingPlanSelector` component to display subscription options
```

Comment:
(Also, just going by the YAML file: in the docs, where will I be able to see how to actually add the SellingPlanSelector? Do we generate a code snippet from the patch, and show it below this step?)